avaframe/com1DFA/com1DFA.py

```diff
@@ -244,20 +250,17 @@ def prepareRelase(cfg, rel, inputSimLines):
     # load info
     entResInfo = inputSimLines['entResInfo']
 
-    # Set release areas and simulation name
+    # Set release areas and release thickness
     relName = rel.stem
     simName = relName
     badName = False
     if '_' in relName:
         badName = True
         log.warning('Release area scenario file name includes an underscore \
         the suffix _AF will be added for the simulation name')
-    releaseLine = inputSimLines['releaseLine']
-    for k in range(len(releaseLine['d0'])):
-        if releaseLine['d0'][k] == 'None':
-            releaseLine['d0'][k] = cfg['GENERAL'].getfloat('relTh')
-        else:
-            releaseLine['d0'][k] = float(releaseLine['d0'][k])
+
+    # set release thickness
+    releaseLine = setThickness(cfg, inputSimLines['releaseLine'], 'useRelThFromIni', 'relTh')
```

Comment:
useRelThFromIni is hardcoded now? Is that wanted?

Reply:
yes, this is the flag that is passed to the function setThickness (which is new - to reduce duplication of code as was almost the same code block for release, secrel and entrainment)